corehq/apps/zapier/views.py

```diff
@@ -86,3 +90,71 @@ def post(self, request, *args, **kwargs):
             return HttpResponse(status=400)
         delete_subscription_with_url(url)
         return HttpResponse('OK')
+
+
+class ZapierCreateCase(View):
+
+    urlname = 'zapier_create_case'
+
+    @method_decorator(csrf_exempt)
+    def dispatch(self, *args, **kwargs):
+        return super(ZapierCreateCase, self).dispatch(*args, **kwargs)
+
+    def post(self, request, *args, **kwargs):
```

Comment:
What are the ways that Zapier does authentication? This is currently exposed to everyone, and doesn't require a password...